pom.xml

```diff
@@ -14,6 +14,7 @@
 
   <properties>
     <marketplace.duoweb.authentication.version>1.7.0-SNAPSHOT</marketplace.duoweb.authentication.version>
+    <nuxeo.target.version>11.1-SNAPSHOT</nuxeo.target.version>
```

Comment:
Replace with `{nuxeo.distribution.version}` as the build and target version is the same.